### DISCUSSION TITLE
Blog Post: Deep Dive into Auth Store

### DISCUSSION THREAD

**Discussion by altugbakan** [stellar/sorobanathon#78] [General]
Hello again!

I have written another technical blog post about the Auth Store quest. Since the second quest contains lots of details about the workings of the Soroban network, it is quite the read! Please check it out [here](https://dev.to/altug/soroban-quest-auth-store-1aea).

Hope you enjoy it!

**Comment by wildework** [comment]
WOW, you're not kidding, that's a really deep dive. Nice work!

**Reply by altugbakan** [reply]
Glad you enjoyed it!

**Comment by rawritude** [comment]
Really enjoying these! Keep it up!

**Reply by altugbakan** [reply]
Thanks! I am planning to!

**Comment by altugbakan** [comment]
Updated for the new changes on Quest 2!

**Comment by achallagundla** [comment]
Heya @altugbakan  ,
Congratulations! Your contribution to Sorobanathon: First Light has been accepted! 🎉
Your submission may qualify for a reward of 1500 XLM! One last step: to receive that reward, please fill out this [form](https://stellarform.typeform.com/Sorobanathon) completely and accurately.

Please let us know if you have any questions. (You can also message me on discord @ anuxhya)

All the best,
Anuhya

PS: No need to submit the form if you already have, just update me on discord if your wallet address has changed.
